tests/late_joiners.py

```diff
@@ -198,11 +198,10 @@ def run(args):
                         if args.consensus == "pbft"
                         else args.raft_election_timeout / 1000
                     )
-
                     if node.node_id == cur_primary_id and args.consensus == "pbft":
-                        # if pbft suspend the primary for twice the elecetion timeout
+                        # if pbft suspend the primary for > 2 the elecetion timeout
```

Comment:
```suggestion
                        # if pbft suspend the primary for > 2 the election timeout
```

Reply:
For typos, I suggest we use the suggestion feature in GitHub as much as possible. It's often quicker than typing "Typo:" :)